.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: go
 go:
+  - 1.6
```

Comment:
A part of me thinks we should just have `1.6` and `1.x`? I don't have a strong opinion though.

Reply:
🤷‍♀️